servers/v3-proxy-api/src/routes/v3Get.ts

```diff
@@ -12,56 +12,53 @@ import { UserSavedItemsByOffsetArgs } from '../generated/graphql/types';
 import * as Sentry from '@sentry/node';
 import { ErrorCodes, getErrorHeaders } from './errorMapper';
 import { serverLogger } from '@pocket-tools/ts-logger';
+import { checkSchema, validationResult, matchedData } from 'express-validator';
+import { V3GetParams, V3GetSchema } from './validations';
 
 const router: Router = Router();
-//v3 in web repo can support both POST and GET request.
-//proxy need to be backward compatible with both of them
 
-router.get('/', async (req: Request, res: Response) => {
```

Comment:
The only difference between these two methods were the source of the data (query vs. body) and the error prefix. Since you can pass the schema validator location in the validation middleware and pull the parameters from the matched data, the only difference was the error prefix.